tests/testthat/_snaps/pstr.md

```diff
@@ -50,17 +50,17 @@
       
       [[4]]
          value
-      1      0
-      2      0
-      3      1
-      4      0
-      5      0
-      6      1
-      7      0
-      8      0
-      9      1
-      10     0
-      11     0
-      12     1
+      1    0.0
+      2    0.0
+      3    1.0
```

Comment:
This is a bit surprising. Can you please check if the `typeof(value)`  is the same for `xctr*()` and `pstr*()`?

Reply:
I have just checked, they are the same (double).

Reply:
<img width="387" alt="Capture d’écran 2023-05-16 à 13 56 46" src="https://github.com/2DegreesInvesting/tiltIndicator/assets/97972818/29f5818a-d9fb-4d16-a743-44fb80515a99">

Reply:
<img width="452" alt="Capture d’écran 2023-05-16 à 13 57 59" src="https://github.com/2DegreesInvesting/tiltIndicator/assets/97972818/b60e4aa6-6b76-4593-a6db-9fd57258eb45">